package.json

```
@@ -126,7 +126,7 @@
 			"firefoxProfile": "./test/web-ext-profile",
 			"chromiumProfile": "./test/web-ext-profile",
 			"startUrl": [
-				"github.com"
+				"https://github.com"
```

There are more features visible here

```suggestion
				"https://github.com/refined-github/refined-github"
```

---

I'd rather start on GitHub's homepage, it's easier to jump to recent issues/PRs that way.

---

> it's easier to jump to recent issues/PRs that way

I don't see how that's possible

---

Sorry I meant the dashboard :man_facepalming:

---

Isn't there a "Recent activity" section on the left side of dashboard?

---

> Isn't there a "Recent activity" section on the left side of dashboard?

I see 4 links to issues unrelated to RGH, I think RGH’s repo is a better starting point to access RGH’s issues. Maybe even open the issues directly

```suggestion
				"https://github.com/refined-github/refined-github/issues"
```